indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -325,7 +327,13 @@ public void stop()
     return ImmutableList.of();
   }
 
-  public Collection<ZkWorker> getWorkers()
+  @Override
+  public Collection<Worker> getWorkers()
```

Comment:
((spoke in chat)) since io.druid.indexing.overlord.http.OverlordResource#getWorkers already special-cases the RTR, suggest making it get zk workers in that case until a more complete solution can be put in place.